source/draw/gpu/opengl/backend/gles3/Gles3Context.ooc

```diff
@@ -180,26 +180,17 @@ Gles3Context: class extends GLContext {
 	}
 
 	createQuad: func -> Gles3Quad {
-		version(debugGL) { validateStart() }
```

Comment:
No, they should only be wrapped around true OpenGL calls (i.e. with the prefix *gl*).